rosidl_typesupport_cpp/src/type_support_dispatch.hpp

```diff
@@ -51,13 +51,21 @@ get_typesupport_handle_function(
       rcpputils::SharedLibrary * lib = nullptr;
 
       if (!map->data[i]) {
-        char library_name[1024];
-        snprintf(
-          library_name, 1023, "%s__%s",
-          map->package_name, identifier);
-        std::string library_path = rcpputils::find_library_path(library_name);
+        std::string library_name{map->package_name};
+        library_name += "__";
+        library_name += identifier;
```

Comment:
@hidmic no reason to delay a potential `std::bad_alloc` to `rcpputils::find_library_path()` call.